e2e/test/scenarios/dashboard-filters/dashboard-filters-text-category.cy.spec.js

```diff
@@ -27,39 +24,67 @@ import {
 
 import { DASHBOARD_TEXT_FILTERS } from "./shared/dashboard-filters-text-category";
 
+const { ORDERS_ID } = SAMPLE_DATABASE;
+
 describe("scenarios > dashboard > filters > text/category", () => {
   beforeEach(() => {
     restore();
     cy.signInAsAdmin();
 
-    visitDashboard(ORDERS_DASHBOARD_ID);
-
-    editDashboard();
+    cy.createQuestionAndDashboard({
+      questionDetails: {
+        query: { "source-table": ORDERS_ID, limit: 5 },
+      },
+      cardDetails: {
+        size_x: 24,
+        size_y: 8,
+      },
+    }).then(({ body: { id, dashboard_id } }) => {
```

Comment:
Cypress was crashing so badly that I had to ease the load by creating a custom dashboard with the 5 row limit in its only card.